web/src/components/tasks/business-formation/review/ReviewStep.tsx

```diff
@@ -32,18 +33,32 @@ export const ReviewStep = (): ReactElement => {
   const hasPurpose = !!state.formationFormData.businessPurpose;
   const hasMembers = (state.formationFormData.members?.length ?? 0) > 0;
   const isCorp = corpLegalStructures.includes(state.formationFormData.legalType);
+  const isNonProfit = state.formationFormData.legalType === "nonprofit";
+
   return (
     <>
       <div data-testid="review-step">
         <ReviewSection stepName={"Business"} testId="edit-business-name-step">
           <BusinessNameAndLegalStructure isReviewStep />
           <ReviewBusinessSuffixAndStartDate />
+          {state.formationFormData.isVeteranNonprofit && (
+            <Content className="margin-top-2">
+              {Config.formation.fields.isVeteranNonprofit.domNpVetCorpReviewText}
+            </Content>
+          )}
           {isForeignCorporation(state.formationFormData.legalType) && (
             <>
               <ReviewWillPracticeLaw willPracticeLaw={state.formationFormData.willPracticeLaw} />
               <ReviewForeignCertificate foreignGoodStandingFile={state.foreignGoodStandingFile} />
             </>
           )}
+          {isNonProfit && <ReviewNonprofitProvisions />}
+          {isNonProfit && hasProvisions && <ReviewProvisions />}
+          {isNonProfit && hasPurpose && (
+            <ReviewSubSection header={Config.formation.fields.businessPurpose.label}>
+              <ReviewText fieldName={"businessPurpose"} isExpandable={true} />
+            </ReviewSubSection>
+          )}
```

Comment:
I'd personally prefer these to be grouped together rather than redoing the same logic on every line:
```
{isNonprofit && (
  <>
    <ReviewNonprofitProvisions />
    {hasProvisions && <ReviewProvisions />}
    ....
  </>
)}
```

Reply:
Changed